ServerlessApplication/ServiceCloudVoiceLambdas.yaml

```diff
@@ -203,8 +197,8 @@ Resources:
           ACCESS_TOKEN_PARAM_NAME: !Sub ${SalesforceRestApiAccessTokenSSMParamName}
           SUBJECT: !Sub ${SalesforceRestApiSubjectSSMParamName}
           AUDIENCE: !Sub ${SalesforceRestApiAudienceSSMParamName}
-          SALESFORCE_AUTH_ENDPOINT: !Sub ${SalesforceAuthEndpoint}
-          SALESFORCE_REST_API_ENDPOINT_BASE: !Sub ${SalesforceRestApiEndpointBase}
+          SALESFORCE_AUTH_ENDPOINT: https://login.salesforce.com/services/oauth2/token
+          SALESFORCE_REST_API_ENDPOINT_BASE: https://login.salesforce.com/services/data/v54.0
```

Comment:
@jinalkathiara  ok to hardcode this endpoint?

Reply:
Yes this is a placeholder param value. Customers are anyway going to update it.